openandroidinstaller/assets/configs/lancelot.yaml

```diff
@@ -0,0 +1,53 @@
+metadata:
+  maintainer: A non (anon)
+  brand: xiaomi
+  device_name: Xiaomi Redmi 9 / Poco M2
+  is_ab_device: false
+  device_code: lancelot
+  supported_device_codes:
+    - lancelot
+    - galahad
+    - shiva
+  untested: true
```

Comment:
Looking at the installations instructions of lineageos, we might want to include some warnings about the correct firmware version here. Did the correct firmware version or starting with the stock rom play a role in your work with Xiaomi devices?

Reply:
For my Xiaomi devices (lavender, pine, ...), the only need is to be on android 10+ (because partitions are not the same iirc)
If the device is old, the latest firmware is the best, else it depends on the ROM (for example rosemary)

Reply:
Maybe we should add a more detailed explanation instead of "You need Android ...", and it can be related to #352 (and maybe `brand` metadata too)

Reply:
Yeah, that makes sense :thinking: